src/Extractor.ts

```diff
@@ -2131,6 +2213,38 @@ class Extractor {
     return this.gql.name(id, id.text);
   }
 
+  externalModule(
+    node: ts.DeclarationStatement,
+    name: NameNode,
+    kind: NameDefinition["kind"],
+  ) {
+    const tag = this.findTag(node, EXTERNAL_TAG);
```

Comment:
Nit: In most places we check for the tag before calling this. Can we pass the external tag as an argument here and avoid this error case?